src/platform/DefaultPlatformFunctions.ts

```diff
@@ -38,6 +41,10 @@ class DefaultPlatformFunctions {
     }
     return result === 0;
   }
+
+  createEmitter(): StripeEmitter | EventEmitter {
+    throw new Error('createEmitter not implemented');
```

Comment:
Feels like this should return a new instance of StripeEmitter

Reply:
Updated in latest commit - `createEmitter` returns `StripeEmitter` in `WebPlatformFunctions`